scripts/test

```diff
@@ -26,9 +26,9 @@ function run_linters() {
     fi
 
     # Lint Python scripts.
-    ./.venv/bin/flake8 --exclude ./*.pyc,./.venv
+    ./.venv/bin/flake8 --exclude "*.pyc,.eggs,.venv"
     if command -v ./.venv/bin/black > /dev/null; then
-        ./.venv/bin/black --check --target-version py36
+        ./.venv/bin/black --check --diff --target-version py36 .
```

Comment:
`black` wasn't actually working because it wasn't given a path. I've set it to check the whole repository and output the diff, like we do [here](https://github.com/azavea/stac-packer/blob/2286e62c4b7d4a35271698a2f337f5f345418e63/scripts/test#L29).

Reply:
Yup! It just said it had nothing to do and returned `0`.